lib/shelley/src/Cardano/Wallet/Shelley/Compatibility/Parser.hs

```diff
@@ -0,0 +1,75 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+{-# LANGUAGE ExistentialQuantification #-}
+
+module Cardano.Wallet.Shelley.Compatibility.Parser
```

Comment:
```suggestion
module Cardano.Wallet.Shelley.Compatibility.ParserTx
```

Not sure.  🤔  This would make it more visible that the module is fairly specialized to Txs for now — and will likely remain that way in the future.

Reply:
Now that I think about it again, I think that the highly probable `TxCBOR` module would be a good place to put these function into. 🤔